extend.php

```diff
@@ -1,16 +1,33 @@
 <?php
 
+/*
+ * This file is part of fof/nightmode.
+ *
+ * Copyright (c) 2020 FriendsOfFlarum.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 namespace FoF\NightMode;
 
 use Flarum\Extend;
+use FoF\Components\Extend\AddFofComponents;
+use FoF\Extend\Extend as FoFExtend;
 use Illuminate\Contracts\Events\Dispatcher;
 
 return [
+    new AddFofComponents(),
     (new Extend\Frontend('forum'))
         ->css(__DIR__.'/resources/less/forum/extension.less')
         ->js(__DIR__.'/js/dist/forum.js'),
+    (new Extend\Frontend('admin'))
+        ->js(__DIR__.'/js/dist/admin.js'),
     new Extend\Locales(__DIR__.'/resources/locale'),
-        function (Dispatcher $events) {
-            $events->subscribe(Listeners\Preferences::class);
-        },
+    (new FoFExtend\ExtensionSettings())
```

Comment:
Looks like you are missing a backslash here between FoF and Extend :)

Should read as `(new FoF\Extend\ExtensionSettings())`